impl/src/main/kotlin/com/android/declarative/internal/DeclarativePlugin.kt

```diff
@@ -41,6 +41,19 @@ class DeclarativePlugin @Inject constructor(
         val cache = DslTypesCache()
         parseDeclarativeBuildFile(project, issueLogger, project.layout.projectDirectory.file(buildFileName), cache)
 
+        // this is a hack until https://github.com/gradle/gradle/issues/22468 is fixed.
```

Comment:
I would separate workarounds in some method for isolation

Reply:
good idea, I moved them to a class called GradleIssuesWorkaround.